code/controllers/subsystem/non_firing/persistent_data.dm

```diff
@@ -13,13 +13,25 @@ SUBSYSTEM_DEF(persistent_data)
 	flags = SS_NO_FIRE
 	/// List of atoms registered into the subsystem for persistent data storage. Can be anything at all
 	var/list/registered_atoms = list()
+	/// Set to true after a the end of the round to prevent griefing being saved
+	var/end_of_round_shutdown = FALSE
```

Comment:
I particularlly set it up this way so that if a reboot happened then was cancelled the data could still be saved at the end of the round... but I don't think that could ever actually happen. I'll make the changes